src/arm/cache.c

```diff
@@ -1,11 +1,15 @@
 #include <stdint.h>
+#include <stdio.h>
 
 #include <cpuinfo.h>
 #include <cpuinfo/internal-api.h>
 #include <cpuinfo/log.h>
 #include <arm/api.h>
 #include <arm/midr.h>
 
+#define STRINGIFY(token) #token
+#define CACHESIZE_FILENAME_FORMAT "/sys/devices/system/cpu/cpu%" PRIu32 "/cache/index%" PRIu32 "/size"
+#define CACHESIZE_FILENAME_SIZE (sizeof("/sys/devices/system/cpu/cpu" STRINGIFY(UINT32_MAX) "/cache/index%" STRINGIFY(UINT32_MAX) "/size"))
```

Comment:
This feels like a wrong way to put those changes, as "/sys" filesystem make sense only for Linux OS